include/swift/Basic/LangOptions.h

```diff
@@ -319,6 +321,9 @@ namespace swift {
     /// to the Swift language version.
     version::Version cxxInteropCompatVersion;
 
+    void setCxxInteropFromArgs(llvm::opt::ArgList &Args,
```

Comment:
I think ideally this would go somewhere like `ArgsToLangOptionsConverter` (similarly to the existing `ArgsToFrontendOptionsConverter`), but I don't think that necessarily has to happen in this patch.